apps/fulltextsearch.sh

```diff
@@ -83,37 +76,66 @@ then
 fi
 
 # Check that ingest-attachment is properly installed
-if ! [ "$(curl -s http://127.0.0.1:9300)" ]
+if ! [ "$(curl -s http://127.0.0.1:9200)" ]
 then
 msg_box "Installation failed!
 Please report this to $ISSUES"
     exit 1
 fi
 
 # Install ReadOnlyREST
-# TODO Check with SHA
+echo "Downloading readonlyrest..."
+rm -f "/tmp/readonlyrest-1.16.15_es$ES_VERSION.zip"
+wget -q -T 10 -t 2 "https://github.com/nextcloud/vm/raw/master/apps/fulltextsearch-files/readonlyrest-1.16.15_es$ES_VERSION.zip" -P /tmp
+mkdir -p "$GPGDIR"
+wget -q -T 10 -t 2 "https://raw.githubusercontent.com/nextcloud/vm/master/apps/fulltextsearch-files/readonlyrest-1.16.15_es$ES_VERSION.zip.sha1" -P "$GPGDIR"
+echo "Verifying checksums..."
+sha1sum /tmp/readonlyrest-1.16.15_es"$ES_VERSION".zip | awk '{print $1}' > "$GPGDIR"/verify1
+cat "$GPGDIR"/readonlyrest-1.16.15_es"$ES_VERSION".zip.sha1 > "$GPGDIR"/verify2
+if [ -z "$(diff $GPGDIR/verify1 $GPGDIR/verify2)" ]
+then
+    echo "Checksum OK!"
+else
+msg_box "Checksum was not OK.
+
+Please report this to $ISSUES."
+rm -rf "$GPGDIR"
+rm -f /tmp/fulltextsearch-files/readonlyrest-1.16.15_es"$ES_VERSION".zip
+exit 1
+fi
+
 if [ -d /usr/share/elasticsearch ]
 then
     cd /usr/share/elasticsearch/bin
-    check_command ./elasticsearch-plugin install "$APP"/fulltextsearch-files/readonlyrest-1.16.15_es"$ES_VERSION".zip
+    check_command ./elasticsearch-plugin file:///tmp/fulltextsearch-files/readonlyrest-1.16.15_es"$ES_VERSION".zip
```

Comment:
Here is missing `install`
`check_command ./elasticsearch-plugin install file:///tmp/fulltextsearch-files/readonlyrest-1.16.15_es"$ES_VERSION".zip`